cmd/account_cmd.go

```diff
@@ -424,3 +435,43 @@ func accountImport(ctx *cli.Context) error {
 	fmt.Printf("\nImport wallet:%s to %s complete, total:%d success:%d failed:%d skip:%d\n", source, target, total, succ, fail, skip)
 	return nil
 }
+
+func accountExport(ctx *cli.Context) error {
+	if ctx.NArg() <= 0 {
+		return fmt.Errorf("Missing target file name")
+	}
+	target := ctx.Args().First()
+	client, err := common.OpenWallet(ctx)
+	if err != nil {
+		return err
+	}
+	wallet := client.GetWalletData()
+	if ctx.IsSet(utils.GetFlagName(utils.AccountLowSecurityFlag)) {
+		n := client.GetAccountNum()
+		passwords := make([][]byte, n)
+		for i := 0; i < n; i++ {
+			ac := client.GetAccountMetadataByIndex(i + 1)
+			fmt.Printf("Account %d %s: %s", i+1, ac.Label, ac.Address)
+			for j := 0; j < 3; j++ {
+				pwd, err := password.GetPassword()
+				if err != nil {
+					fmt.Println(err)
+				} else {
+					passwords[i] = pwd
+					break
+				}
+			}
+		}
+		wallet = wallet.Dump()
+		err := wallet.ToLowSecurity(passwords)
+		if err != nil {
+			return fmt.Errorf("export failed: %s", err)
+		}
+	}
```

Comment:
if not to low security, to default security? or to original security?

Reply:
Same as the original wallet.